package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "okunokumo.be",
     "version": "1.0.0",
-    "description": "",
+    "description": "All 2 test suites and 10 tests passed with no errors, achieving 71.61% test coverage. Notable successes include passing all authentication tests and successfully adding and approving reviews.",
```

Comment:
Nah you dont have to do this
I meant on description of your PR in Github haha

Reply:
Oh alright my bad haha removinggg